neo/SmartContract/InteropService.NEO.cs

```diff
@@ -40,6 +42,8 @@ static partial class InteropService
         public static readonly uint Neo_Iterator_Keys = Register("Neo.Iterator.Keys", Iterator_Keys, 0_00000400);
         public static readonly uint Neo_Iterator_Values = Register("Neo.Iterator.Values", Iterator_Values, 0_00000400);
         public static readonly uint Neo_Iterator_Concat = Register("Neo.Iterator.Concat", Iterator_Concat, 0_00000400);
+        public static readonly uint Neo_Json_Deserialize = Register("Neo.Json.Deserialize", Json_Deserialize, 100_00000000/*TODO: Compute gast cost*/);
+        public static readonly uint Neo_Json_Serialize = Register("Neo.Json.Serialize", Json_Serialize, 100_00000000/*TODO: Compute gast cost*/);
```

Comment:
The cost can be the same as `System.Runtime.Serialize` and `System.Runtime.Deserialize`.